app/uk/gov/hmrc/gform/gform/FormController.scala

```diff
@@ -326,17 +326,25 @@ class FormController(
         def goBack(toSectionNumber: SectionNumber) = {
           val formModel = formModelOptics.formModelRenderPageOptics.formModel
           val sectionTitle4Ga = sectionTitle4GaFactory(formModel(toSectionNumber), sectionNumber)
-          Redirect(
-            routes.FormController
-              .form(
-                cache.formTemplateId,
-                maybeAccessCode,
-                toSectionNumber,
-                sectionTitle4Ga,
-                SuppressErrors.Yes,
-                ff
-              )
-          ).pure[Future]
+          ff match {
+            case FastForward.CYA(_, _) =>
+              Redirect(
+                routes.SummaryController
+                  .summaryById(cache.formTemplateId, maybeAccessCode, sectionNumber.toCoordinates, None)
```

Comment:
I think we should got to this route only for 
```
case FastFrorward.CYA(_, None)
```
If the second argument is `Some(sectionNumber)` it means that it is either ATL CYA pager or mini summary list CYA page

Reply:
I think we should go to this `sectionNumber` instead of the final CYA page